src/groups/bmq/bmqa/bmqa_session.h

```diff
@@ -679,6 +680,13 @@ class Session : public AbstractSession {
   public:
     // TYPES
 
+    /// Pool of shared pointers to Blobs
```

Comment:
We don't expose `bmqp` package in `bmqa`, since `bmqa` is a client package. And we need to declare `BlobSpPool` somewhere to be used in `bmqa::MockSession`. I moved it directly to private types in `bmqa::MockSession`